extensions/Worker.Extensions.ServiceBus/src/ServiceBusReceivedMessageConverter.cs

```diff
@@ -0,0 +1,41 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.IO;
+using System.Threading.Tasks;
+using Azure.Messaging.ServiceBus;
+using Azure.Messaging.ServiceBus.Primitives;
+using Microsoft.Azure.Functions.Worker.Converters;
+using Microsoft.Azure.Functions.Worker.Core;
+
+namespace Microsoft.Azure.Functions.Worker;
+
+internal class ServiceBusReceivedMessageConverter : IInputConverter
+{
+    public ValueTask<ConversionResult> ConvertAsync(ConverterContext context)
+    {
+        if (context is null)
+        {
+            return new ValueTask<ConversionResult>(ConversionResult.Unhandled());
+        }
+
+        return context.Source switch
```

Comment:
As per Fabio's comment on the blob PR, you can remove the context null check if-statement

>Having the context being null here is not expected, if validating, you should throw if this is the case. Otherwise, the check here would be redundant if you change your switch to use context?.Source, so you can consider doing that.

```suggestion
        return context?.Source switch
```

Reply:
According to the nullable references, context can't even be null so the `?` is redundant.